src/App.tsx

```diff
@@ -1,39 +1,156 @@
-import { useState } from 'react';
-import { Routes, Route } from 'react-router-dom';
 import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
 import { utils } from 'ethers';
+import { useState } from 'react';
+import { Route, Routes } from 'react-router-dom';
 import errorIcon from './assets/icons/error.svg';
+import loadingIcon from './assets/icons/loading.svg';
+import successIcon from './assets/icons/success.svg';
 import { AppDialog } from './components/AppDialog/AppDialog.component';
+import { DateSelector } from './components/DateSelector/DateSelector.component';
 import { Header } from './components/Header/Header.component';
 import { InputWithButtonPillGroup } from './components/InputPillGroup/InputWithButtonPillGroup.component';
 import { Urls } from './constants';
 import { DashboardContainer } from './pages/Dashboard/Dashboard.container';
 import { ProposalDetailsContainer } from './pages/ProposalDetails/ProposalDetails.container';
 import { ProposalsListContainer } from './pages/ProposalsList/ProposalsList.container';
 import { StakingContainer } from './pages/Staking/Staking.container';
-import { DateSelector } from './components/DateSelector/DateSelector.component';
+
+type MintingProcessInfo = {
+  label: string;
+  value: string;
+  isProminant?: boolean;
+};
 
 function App() {
   const [selectedDate, setSelectedDate] = useState<number>();
-  const [isOpenDialog, setIsOpenDialog] = useState(false);
+  const [openDialogType, setOpenDialogType] = useState<string | null>(null);
+
+  const dialogVals: MintingProcessInfo[] = [
```

Comment:
Both success and progress dialogs shows the same information. What do you think about creating a common component that will contain info about the deposit and than use it in the content for both of them? 
![image](https://user-images.githubusercontent.com/49481475/154922465-45a6cc74-9fc6-47d8-b2e2-afffc5d41a18.png)